cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -112,34 +118,38 @@ Running Servers with SSL
 
 To enable running servers with SSL in CDAP, add this property to ``cdap-site.xml``:
 
-================================================= ===========================================================================
-Property                                          Value
-================================================= ===========================================================================
-``ssl.enabled``                                   ``true``
-================================================= ===========================================================================
+================================================= ==================== ======================================================
+Property                                          Default Value        Description
+================================================= ==================== ======================================================
+``ssl.enabled``                                   ``true``             ``true`` to enable servers running with SSL in CDAP
+================================================= ==================== ======================================================
 
 Default Ports
 .............
 
 Without SSL:
 
-================================================= ===========================================================================
-Property                                          Default Value
-================================================= ===========================================================================
-``router.bind.port``                              ``10000``
-``security.auth.server.bind.port``                ``10009``
-``dashboard.bind.port``                           ``9999``
-================================================= ===========================================================================
+================================================= ==================== ======================================================
+Property                                          Default Value        Description
+================================================= ==================== ======================================================
+``router.bind.port``                              ``10000``            Port number that the CDAP Router should bind to for 
+                                                                       HTTP Connections
+``security.auth.server.bind.port``                ``10009``            Port number that the CDAP Authentication Server should
+                                                                       bind to for HTTP
+``dashboard.bind.port``                           ``9999``             CDAP Console bind port
```

Comment:
should we make this consistent with the others? "Port number that the CDAP Console should bind to for HTTP Connections" ?